modules/ROOT/pages/storage.adoc

```diff
@@ -354,6 +354,70 @@ storage:
       with_mount_unit: true
 ----
 
+== Adding Network Storage
+
+Fedora CoreOS systems can be configured to mount network file systems such as NFS and CIFS. This is best achieved by using Ignition to create systemd units. File systems can be mounted on boot by creating a standard mount unit. Alternately, a file system can be mounted when users access the mountpoint by creating an additional automount unit. Below are examples of each for an NFS file system.
+
+=== Configuring NFS Mounts
+
+Creating a systemd unit to mount an NFS file system on boot. Note: the .mount file must be named based on the path (e.g. /var/mnt/data = var-mnt-data.mount)
```

Comment:
The leading periods on these titles got dropped, so they're no longer being treated as titles of the examples.

The Note could be moved to a callout using [admonition](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#admonitions) syntax.

`.mount`, `/var/mnt/data`, and `var-mnt-data.mount` should probably all be monospaced.

Reply:
Added Admonition syntax.